lib/index.ts

```diff
@@ -27,26 +41,27 @@ export function inspect(root, targetFile, options) {
     baseargs = ['run', 'python'];
   }
 
-  return Promise.all([
+  const [plugin, pkg] = await Promise.all([
```

Comment:
```suggestion
  const [plugin, package] = await Promise.all([
```
so that you can do
```
  return { plugin, package };
```

Reply:
`package` is a reserved word in Javascript.